net/Socket.cpp

```diff
@@ -1303,13 +1306,18 @@ bool StreamSocket::parseHeader(const char *clientName,
 {
     assert(map._headerSize == 0 && map._messageSize == 0);
 
+    constexpr std::chrono::duration<float, std::milli> delayMax =
+        std::chrono::duration_cast<std::chrono::milliseconds>(SocketPoll::DefaultPollTimeoutMicroS);
```

Comment:
I expect we'd want a separate value for the `delayMax`, other than `DefaultPollTimeoutMicroS`, as I don't see how a socket-specific timeout would be related to our poll-timeout, which can be arbitrarily small or long (indeed, it can even be infinite!).